rpm/unofficial-jolla-language-pack-bg.spec

```diff
@@ -0,0 +1,26 @@
+#make sure change these variables to suit your language
+%define CONFLANG bg
+%define LOCNAME bg_BG
+%define RPM_SUFFIX bg
+%define QM_SUFFIX bg
+
+Name: unofficial-jolla-language-pack-%{CONFLANG}
+Version:	2.1.2
```

Comment:
Hi @omkpuBamev 
It might be useful to set the package version to the same as the SFOS version at Pootle when the lang files retrived.

Reply:
Sure, can I find it somewhere on Pootle? Seems I'm missing it somehow.